### ISSUE TITLE
Russian language turning into crap with vnc connection

### ISSUE THREAD

**Issue by KKostikov** [mRemoteNG/mRemoteNG#1214]
Hello.
Russian lletters turning into crap over vnc protocol connection from win10 to win10 (at least).
RDP protocol connection works fine
English language works fine

RNG ver. 1.76.11.40527
Windows 10 Pro (any build)

**Comment by KKostikov** [comment]
Sorry i forgot to mention, we use TightVNC server for Windows ver. 2.8.11 as the server part. We chose this one because it has an msi package for mass deployment in AD structure.  Heres screenshot
![mremoteng](https://user-images.githubusercontent.com/36406800/50326616-5c667900-04fc-11e9-9f2a-a3f7a3c7f970.png)

**Comment by sparerd** [comment]
Thanks @KKostikov. Does this happen immediately after connecting, or does it work fine for a while and then start showing those odd characters?

**Comment by KKostikov** [comment]
Immediately when we switch to Russian language.

пт, 21 дек 2018, 17:40 David Sparer notifications@github.com:

**Comment by l-luk** [comment]
I approved that bug.

**Comment by nagorsky** [comment]
Same issue on 1.76.20.24615   and last version of TightVNC.
Can anybody fix it?

**Comment by Gibrid89** [comment]
Yes, on 1.76.20.24615 and TightVNC russian language turning into crap...

**Comment by Kvarkas** [comment]
will check possibly issue on TightVNC side by need verification

**Comment by Kvarkas** [comment]
I am able to reproduce such, will think how we can fix that - problem is that keypress don't send valid key code for some reason
